src/sdk/eng/Signing.props

```diff
@@ -84,6 +84,14 @@
   
   <ItemGroup>
     <FileSignInfo Include="StreamJsonRpc.dll" CertificateName="MicrosoftSHA2" />
+
+    <!-- Roslyn apphosts -->
+    <FileSignInfo Include="csc" CertificateName="$(DotNetCertificateName)" />
+    <FileSignInfo Include="vbc" CertificateName="$(DotNetCertificateName)" />
+    <FileSignInfo Include="VBCSCompiler" CertificateName="$(DotNetCertificateName)" />
+    <FileSignInfo Include="csc.exe" CertificateName="$(DotNetCertificateName)" />
+    <FileSignInfo Include="vbc.exe" CertificateName="$(DotNetCertificateName)" />
+    <FileSignInfo Include="VBCSCompiler.exe" CertificateName="$(DotNetCertificateName)" />
```

Comment:
.exe should be signed by default due to arcade's defaults, I think you can remove this.